client/src/components/App.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import axios from 'axios';
+import ListOfHomes from './ListOfHomes.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: []
+    }
+  }
+
+  componentDidMount(e) {
+    var that = this;
```

Comment:
Hi Erzhena, Is that variable 'that' necessary on line 15? Can you just use 'this' instead?

Reply:
Fixed it!